### PR TITLE
Normalize only when there is no decimal points

### PR DESCRIPTION
**Pull request by szewczykmira**

### CODE REVIEW

File: django_prices/templatetags/prices.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
 
 def normalize_price(price, normalize):
-    if normalize:
+    _, value = divmod(price, 1)
```
Comment (krzysztofwolski) [inline django_prices/templatetags/prices.py]: Renaming `value` to `decimals` would help with readability

**Comment by patrys** [comment]
Why do we want/need this? What's the point of calling normalize if there is no decimal point?

**Comment by szewczykmira** [comment]
@patrys Because one of our clients wants to cut decimal points only when decimal points are equal to 0.

**Comment by patrys** [comment]
Ok, I misunderstood the intent. In this case can't we call normalize and then inspect the result instead of calling divmod? It feels like a workaround.

**Comment by szewczykmira** [comment]
@patrys can you check if it looks okey?

**Pull request merged**